okta/okta-signin-widget
# Security Question verify and testcafe test

## Description:

- update authenticator object structure
- also pull latest idxjs
- add verify SQ page
- add test for SQ enroll and verify

## PR Checklist

- [ ] Have you verified the basic functionality for this change?



- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Screenshot/Video:

![hw-authenticator-api-sq](https://user-images.githubusercontent.com/13139863/84552956-60c1bb80-acc7-11ea-8ee1-b1c6fbcebc99.gif)

### Reviewers:


### Issue:

- [OKTA-304275](https://oktainc.atlassian.net/browse/OKTA-304275)
